contracts/test/utils.ts

```diff
@@ -1,4 +1,22 @@
 import { expect } from 'chai'
+import fs from 'fs'
+import { json } from 'starknet'
+
+export const loadStarkgateContract = (name: string): any => {
```

Comment:
Is this whole folder now redundant? https://github.com/smartcontractkit/chainlink-starknet/tree/develop/packages-ts/integration-starkgate